migrations/v2.js

```diff
@@ -0,0 +1,115 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+const getCourse = content => {
+  const course = content.find(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._extensions?._pageLevelProgress;
+};
```

Comment:
As you're setting coursePlpGlobals within the describe function might be a bit simpler to remove getGlobals and just use coursePlpGlobals.